smartsim/log.py

```diff
@@ -129,3 +129,23 @@ def log_to_file(filename: str, log_level: str = "debug") -> None:
         filename, "w+", encoding="utf-8"
     )
     coloredlogs.install(stream=stream, logger=logger, level=log_level)
+
+
+def log_to_file_preview(
+    filename: str,
+    logger: logging.Logger,
+) -> logging.Handler:
+    """Installs a filestream handler to the root logger,
+    allowing logging calls to be sent to filename for preview.
+    :param filename: the name of the desired preview file.
+    :type filename: str
+    """
+    logger.propagate = False
+    fmt = None
+    formatter = logging.Formatter(fmt)
```

Comment:
Do you really want a None here? would `"%(message)s"` do the job? 

I don't even know what happens if you have a null formatter!